opencog/guile/SchemeSmobValue.cc

```diff
@@ -144,9 +145,8 @@ SCM SchemeSmob::ss_new_value (SCM stype, SCM svalue_list)
 	{
```

Comment:
This is currently OK as is, but in the future, you will probably need to change this to
```
  if (nameserver().isA(t, FLOAT_VALUE)
  { ...
```